template/template.go

```diff
@@ -1,46 +1,124 @@
 package template
 
 import (
+	"strings"
+
 	"github.com/hashicorp/hil"
 	"github.com/hashicorp/hil/ast"
+	"github.com/openpixel/rise/config"
 	"github.com/openpixel/rise/interpolation"
 )
 
 // Template is a container for holding onto the ast Variables
 type Template struct {
```

Comment:
Now that we have an object of type "Template" part of config, it seems confusing to also have this object. I'm going to consider renaming this.